desc/profiles.py

```diff
@@ -687,6 +693,102 @@ def from_values(cls, x, y, order=6, rcond=None, w=None, sym="auto", name=""):
         return cls(params, sym=sym, name=name)
 
 
+class TwoPowerProfile(_Profile):
+    """Profile represented by two powers.
+
+    f(x) = a[0]*(1 - x**a[1])**a[2]
+
+    Notes
+    -----
+    df/dx = inf at x = 0 if a[1] < 1
+    df/dx = inf at x = 1 if a[2] < dr
+
+    Parameters
+    ----------
+    params: array-like
+        Coefficients of the two power formula. Must be an array of size 3.
+        Default if not specified is [0, 1, 1].
+    name : str
+        Name of the profile.
+
+    """
+
+    _io_attrs_ = _Profile._io_attrs_
+
+    def __init__(self, params=None, name=""):
+        super().__init__(name)
+
+        if params is None:
+            params = [0, 1, 1]
+        self._params = np.atleast_1d(params)
+
+        errorif(
+            self._params.size != 3, ValueError, "params must be an array of size 3."
+        )
+        warnif(
```

Comment:
add why this is occuring (a[1]<1) in the warning string, I know this in the docstring but the warning is what will pop up first when this is encountered